Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -830,24 +817,25 @@ class DropDownMenu
     return item;
   }
 
-  private static OnItemClick (this: HTMLAnchorElement, ev: MouseEvent): boolean
+  private static OnItemClick (anchor: HTMLAnchorElement, ev: MouseEvent, clickHandler: Nullable<Function>): boolean
   {
-    ev.preventDefault();
-
-    if (this.href == null || this.href === '')
+    if (anchor.href == null || anchor.href === '')
```

Comment:
```suggestion
    if (!anchor.href)
```

Reply:
@a-ctor I'll make this with the `===` compare instead to make the code easier to read. you need to know more about Javascript behavior with collapsing null and empty.